src/logic/operate.js

```diff
@@ -0,0 +1,16 @@
+const Big = require('big.js');
+
+function operate(num1, num2, operator) {
+  const [a, b] = [Big(num1), Big(num2)];
+  let result;
+
+  if (operator === '-') result = a.minus(b);
+  if (operator === '+') result = a.plus(b);
+  if (operator === '÷') result = a.div(b);
```

Comment:
Dont forget to manage division by ZERO on milestone 5

Reply:
Thanks for the tip, will do!